pages/get-started/configure-your-database/postgres/aws-rds-directions.mdx

```diff
@@ -6,50 +6,57 @@ import { Callout } from 'nextra/components'
 If you're using AWS RDS Postgres, you'll need to make a few configuration changes
 via the AWS console to enable ReadySet to properly replicate tables.  
 
+Before each step listed below, you'll need to:
+
+- Log in to your AWS account.
+- Click on RDS, which can be found by searching for it in the search bar, under the Services menu, or in your Recently Visited area. 
+
 ## Instructions
 ### 1. Establish a network connection
 <Callout type="info"> Note: these directions assume you are using ReadySet in a different VPC than the upstream database you're connecting to. If ReadySet is within the same VPC as your database, you can ignore this step.</Callout>
 
 To make sure ReadySet can connect with your database, the database will have to be publicly accessible.
```

Comment:
@zoready -- see my note in Slack. I'm hoping to change this to something like:

To make sure ReadySet can connect with your database, it’s easiest if you install ReadySet in the same VPC as your database. If ReadySet is located in another VPC, you can set up VPC peering; otherwise, the database will have to be publicly accessible.